CONTRIBUTING.md

```diff
@@ -98,12 +95,6 @@ and run the `npm run test` command. Keep in mind that this will also run linters
 
```

Comment:
Note:  I am not able to run `gulp eslint` or `lintJS` as defined in lines 78-80. Wanted to flag in case it has anything to do with the `gulp watch` error I see in watch.js below

```sh
gulp eslint
gulp lintSass
gulp lintJS
```

Reply:
Fixed in 746ca4e6f99f48dec570749e5bdd3f611d4daac8.